lib/css/res.scss

```diff
@@ -679,8 +679,50 @@ div.usertext-edit {
 }
 
 .toggleButton {
+
+	& .toggleThumb {
+			cursor: pointer;
+			top: 0;
+			left: 0;
+			right: 0;
+			bottom: 0;
+			width: 40px;
+			height: 10px;
+			background-color: #ccc;
+			border: 2px solid #ccc;
+			-webkit-transition: .4s;
+			-ms-transition: .4s;
+			transition: .4s;
+	}
+
+	& .toggleThumb:before {
+		position: absolute;
+		content: "";
+		height: 10px;
+		width: 10px;
+		left: 2px;
+		background-color: white;
+		-webkit-transition: .4s;
+		-ms-transition: .4s;
+		transition: .4s;
+	}
+
+	&.enabled .toggleThumb:before {
+		-webkit-transform: translateX(30px);
+		-ms-transform: translateX(30px);
+		transform: translateX(30px);
+	}
+
+	&.enabled::after {
+		content: '\00a0On';
```

Comment:
Needs overrides for i18n and custom text. Maybe:

In `settingsConsole.js` or something:

```javascript
addCSS(`
    .toggleButton::after { content: '${i18n('toggleDisabledText')}'; }
    .toggleButton.enabled::after { content: '${i18n('toggleEnabledText')}'; }
`);
```

```scss
/* Custom toggle labels */
&[data-enabled-text]::after {
    content: '\00a0' attr(data-enabled-text);
}

&.enabled[data-enabled-text]::after {
    content: '\00a0' attr(data-enabled-text);
}
```

and in `createElement`, set `onText` and `offText` to those data attributes (using `HTMLElement.setAttribute`, not `$.fn.data()`